### PR TITLE
Changes to upgrade chef client support from version: 12.15+ to version: 15.3+

### PR DESCRIPTION
**Pull request by RajeshPaul38**
Signed-off-by: Rajesh Paul <rajesh.paul@progress.com>

# Description

Describe what this change achieves

## Issues Resolved

List any existing issues this PR resolves

## Check List

- [x] A summary of changes made is included in the CHANGELOG under `## Unreleased`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by RajeshPaul38** [comment]
@tas50 I've added chef version support as 15.3+

**Comment by RajeshPaul38** [comment]
> Hi @RajeshPaul38
> 
> Could you add an entry into `CHANGELOG.md` under the heading `## Unreleased` please do not create a title on this entry, just documentation of what you have changed?
> 
> Can you also please update the description of this PR from the template text which it currently has there now
> 
> Also while you are looking at this cookbook could you please resolve the warnings in cookstyle: `Use the archive_file resource built into Chef Infra Client 15+ instead of the seven_zip_archive (https://docs.chef.io/workstation/cookstyle/chef_modernize_sevenziparchiveresource)`
> 
> Once the tests are passing we can get this merged in 👍

Hi @Xorima ,
Have added the changes in changelog. But the archive_file doesn't have a source attribute. So not sure what should be the correct replacement. Pls suggest.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This file is used to list changes made in each version of the mingw cookbook.
 
 ## Unreleased
 
-- Changes to upgrade chef client support from version: 12.15+ to version: 15.3+
+- Updated minimum chef client version supported, dropping `< 15.3`
```
Comment (damacus) [inline CHANGELOG.md:7]:
```suggestion
- Updated minimum Chef infra-client version supported, dropping `< 15.3`
```

**Review by damacus — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [4.0.0](https://github.com/sous-chefs/mingw/releases/tag/4.0.0)

**Pull request merged**